docs/app-store-connect/apps/builds.md

```diff
@@ -22,6 +28,32 @@ app-store-connect apps builds [-h] [--log-stream STREAM] [--no-color] [--version
 
 
 Application Apple ID. An automatically generated ID assigned to your app
+### Optional arguments for action `builds`
+
+##### `--build-id=BUILD_ID_RESOURCE_ID_OPTIONAL`
+
+
+Alphanumeric ID value of the Build
+##### `--build-version-number=BUILD_VERSION_NUMBER`
+
+
+Build version number is the version number of the uploaded build. For example `46` or `1.0.13.5`.
+##### `--expired`
+
+
+List only expired builds
+##### `--not-expired`
```

Comment:
Since those are mutually exclusive options I prefer to keep them as boolean switches. Especially because for most use cases user would provide only one. Having it defined as enum valued flag would make the command more execution unneccesary involved. Also, it is validated during runtime that both options are not given at the same time:
```bash
$ priit@groundhog app-store-connect apps builds 1481211155 --expired --not-expired
usage: app-store-connect [-h] [--log-stream {stderr,stdout}] [--no-color] [--version] [-s] [-v]
                            {app-store-version-submissions,app-store-versions,apps,beta-app-review-submissions,beta-build-localizations,beta-groups,builds,create-bundle-id,create-certificate,create-profile,delete-bundle-id,delete-certificate,delete-profile,fetch-signing-files,get-bundle-id,get-certificate,get-latest-app-store-build-number,get-latest-testflight-build-number,get-profile,list-builds,list-bundle-id-profiles,list-bundle-ids,list-certificates,list-devices,list-profiles,publish,register-device}
                            ...
app-store-connect: error: argument --not-expired: Using mutually exclusive switches '--expired' and '--not-expired'.
```

Reply:
Updated descriptions in 34aaf3d.